src/views/ApartmentApp/index.js

```diff
@@ -186,20 +224,26 @@ export default class ApartApp extends Component {
       this.saveButtonAlertTimeout = setTimeout(() => {
         this.saveButtonAlertTimeout = null;
         this.setState({ saving: false });
-      }, 2000);
+      }, 6000);
     }
   }
 
-  handleCloseOkay = () => {
-    this.setState({ submitDialogOpen: false, errorDialogOpen: false, errorDialogText: null });
+  handleCloseSnackbar = (event, reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    this.setState({ snackbarOpen: false });
```

Comment:
Do you not set the snackbarOpen state to false if they clickaway? Are you preventing them from clicking out of the dialog box or do you just not need to set the state to false in the case of clickaway- it closes on its own?

Reply:
This makes it so that the snackbar will not close if the user clicks somewhere else on the page. In other word, it restrict it to only close if the user clicks the close button deliberately or the snackbar automatically times-out after 6 seconds

Reply:
In #911 I added a comment earlier this week that explains the purpose of that line, because it caused the same confusing when Nick reviewed the code

Reply:
Nice, that makes sense, and that comment was a good idea